# editing_development

This contains the supplementary file for the article

A Guide to the Chloroplast Transcriptome Analysis Using RNA-Seq

Michel EJS, Hotto AM, Strickler SR, Stern DB, Castandet B

Methods Mol Biol. 2018;1829:295-313. doi: 10.1007/978-1-4939-8654-5_20.

ABSTRACT
Since its first use in plants in 2007, high-throughput RNA sequencing (RNA-Seq) has generated a vast amount of data for both model and nonmodel species. Organellar transcriptomes, however, are virtually always overlooked at the data analysis step. We therefore developed ChloroSeq, a bioinformatic pipeline aimed at facilitating the systematic analysis of chloroplast RNA metabolism, and we provide here a step-by-step user's manual. Following the alignment of quality-controlled data to the genome of interest, ChloroSeq measures genome expression level along with splicing and RNA editing efficiencies. When used in combination with the Tuxedo suite (TopHat and Cufflinks), ChloroSeq allows the simultaneous analysis of organellar and nuclear transcriptomes, opening the way to a better understanding of nucleus-organelle cross talk. We also describe the use of R commands to produce publication-quality figures based on ChloroSeq outputs. The effectiveness of the pipeline is illustrated through analysis of an RNA-Seq dataset covering the transition from growth to maturation to senescence of Arabidopsis thaliana leaves.

The article can be accessed here 

https://link.springer.com/protocol/10.1007%2F978-1-4939-8654-5_20

The pdf is also available in this repository
